src/components/CreateCredential.js

```diff
@@ -17,53 +20,131 @@ const CreateCredential = () => {
+          <GetCredential.Return.CredentialDetails.APIKey heading="API Key" />
+          <GetCredential.Return.CredentialDetails.AllowedOrigins heading="Allowed domains" />
+          <GetCredential.Return.CredentialDetails.OrganizationName heading="Organization" />
+        </GetCredential.Return.CredentialDetails>
+
+        <GetCredential.Return.ProjectsDropdown label="Projects" subHeading="Only your projects that contain credentials are shown" />
+
+        <GetCredential.Return.ManageDeveloperConsole label="Manage all your projects and credentials on Adobe Developer Console" direction='/console/projects' />
+
+        <GetCredential.Return.DevConsoleLink heading="Developer Console Project" />
+
+        <GetCredential.Return.Products label="Included products and services">
+          <GetCredential.Return.Product label="Adobe Express Embed SDK" icon={creativeCloud} />
+        </GetCredential.Return.Products>
+
+      </GetCredential.Return>
+
+      <GetCredential.RequestAccess
+        title="Get credentials"
+        paragraph="Create unique credentials that you will use to call multiple APIs from your application."
+      >
+        <GetCredential.RequestAccess.EdgeCase>
+          <GetCredential.RequestAccess.EdgeCase.NoProduct title="Your organization does not have access to Adobe Express Embed SDK" buttonLabel="Contact us to learn more" buttonLink="#someLink" />
+          <GetCredential.RequestAccess.EdgeCase.NotMember title="Access to Adobe Express Embed SDK APIs is not available at this time." buttonLabel="Learn more about Adobe Express Embed SDK" buttonLink="#someLink" />
+          <GetCredential.RequestAccess.EdgeCase.NotSignUp title="Adobe Express Embed SDK APIs is available as part of the beta program. Sign up for the program or log in to an account that has access." buttonLabel="Sign up for the beta" buttonLink="#someLink" />
+        </GetCredential.RequestAccess.EdgeCase>
+
+        <GetCredential.RequestAccess.RestrictedAccess
+          title="Restricted Access"
+          buttonLabel="Request access"
+        >
+          <GetCredential.RequestAccess.RestrictedAccess.Products label="Included products and services">
+            <GetCredential.RequestAccess.RestrictedAccess.Products.Product label="Adobe Express Embed SDK" icon={creativeCloud} />
+          </GetCredential.RequestAccess.RestrictedAccess.Products>
+        </GetCredential.RequestAccess.RestrictedAccess>
+        <GetCredential.RequestAccess.RequestAccessSide>
+          <div style="display : flex ; gap : 16px ; flex-direction : column;">
+            <h3 className="spectrum-Heading spectrum-Heading--sizeS side-header" >API key credential</h3>
+            <p className="spectrum-Body spectrum-Body--sizeM">An API Key credential was created. The API key credential identifies your application to Adobe servers and can help accept or reject request originating from certain domains.</p>
```

Comment:
Why does it say `An API Key credential was created.`?

Reply:
@ErinFinnegan ,For this too